manifests/nicmode/sriov-node-policy.yaml.j2

```diff
@@ -8,9 +8,11 @@ spec:
   eSwitchMode: "switchdev"
   nicSelector:
     pfNames:
-    - ens1f0
+      {%- for line in pfNamesAll %}
+        - {{ line -}}
```

Comment:
all pf that have an ip? why all?

Reply:
Each node could have a different PF depending where the cards are plugged in.